lib/lib.misc.c

```diff
@@ -28,19 +28,19 @@ void _alert(const char *libdir, const char *msgtype, const char *description)
 {
 	static char none[1] = "";
 	char *cmd;
-	int dummy __attribute__((unused));
+	int demo __attribute__((unused));
```

Comment:
This is clearly not `demo`, but I can't think of what is yet. Placeholder clearly is also wrong. `tmp` would work.